app/views/node.html

```diff
@@ -3,203 +3,285 @@
-              <span class="load-label">15{{ 'NODE.MINUTES' | translate }}</span>
-            </p>
-          </div>
-        </div>
-        <div class="col-sm-2 col-xs-4">
-          <div class="panel">
-            <h3 class="panel-headline">{{ 'NODE.CPU_CORES' | translate }}</h3>
-            <p class="announcement-heading" ng-show="node.num_cores > 0">
-              <span class="load-value">{{ node.num_cores }}</span>
-            </p>
-            <p class="announcement-heading" ng-hide="node.num_cores > 0">-</p>
-            <p class="announcement-text">
-              <span class="load-label">{{ 'NODE.CORES' | translate }}</span>
-            </p>
-          </div>
-        </div>
-        <div class="col-sm-6 col-xs-12">
-          <div class="panel">
-            <h3 class="panel-headline">{{ 'NODE.CRATE_CPU_USAGE' | translate }}</h3>
-            <div class="chart-container">
-              <nvd3-multi-bar-horizontal-chart id="chart-process-cpu"
-                data="processCpuData"
-                margin="{left: 0, top: 0, bottom: 0, right: 0}"
-                stacked="true"
-                tooltips="true"
-                tooltipcontent="toolTipUsedPercentFunction()">
-                <svg style="height: 20px"></svg>
-              </nvd3-multi-bar-horizontal-chart>
-            </div>
-          </div>
+    <div class="node-detail__row">
+      <div class="node-detail__graph__wrapper">
+        <div class="cr-panel-block__item__header">
+          {{ 'NODE.CRATE_CPU_USAGE' | translate }}
+        </div>
+        <div class="node-detail__graph__container cr-panel--default">
+          <nvd3-multi-bar-horizontal-chart id="chart-process-cpu"
+            data="processCpuData"
+            margin="{left: 0, top: 0, bottom: 0, right: 0}"
+            stacked="true"
+            tooltips="true"
+            tooltipcontent="toolTipUsedPercentFunction()">
+            <svg style="height: 20px"></svg>
+          </nvd3-multi-bar-horizontal-chart>
         </div>
       </div>
+    </div>
+  </div>
+
+  <div class="node-detail__subsection">
+    <div class="node-detail__subsection--small">
+      <h2 class="">{{ 'NODE.LOAD' | translate }}</h2>
+      <div class="node-detail__subsection__detail cr-panel-block">
+        <div class="node-detail__subsection__cell--small">
+          <p class="cr-panel-block__item__header ng-binding">
+            1{{ 'NODE.MINUTES' | translate }}
+          </p>
+          <p class="cr-panel-block__item__content--full-width cr-panel--default" ng-show="node.load['1'] >= 0">
+            {{ node.load['1']|number:2 }}
+          </p>
+        </div>
+
+        <div class="node-detail__subsection__cell--small">
+          <p class="cr-panel-block__item__header ng-binding">
+            5{{ 'NODE.MINUTES' | translate }}
+          </p>
+          <p class="cr-panel-block__item__content--full-width cr-panel--default" ng-show="node.load['1'] >= 0">
```

Comment:
Only because in the original in master, the logic that determined whether any of them were displayed at all was this. See https://github.com/crate/crate-admin/blob/master/app/views/node.html#L100